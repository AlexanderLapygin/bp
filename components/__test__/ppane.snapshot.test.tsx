import * as React from "react";
import * as TestRenderer from 'react-test-renderer';
import Panel from "../PPane";


it('renders correctly', () => {
    const tree = TestRenderer.create(<Panel />).toJSON()
    expect(tree).toMatchSnapshot()
})
