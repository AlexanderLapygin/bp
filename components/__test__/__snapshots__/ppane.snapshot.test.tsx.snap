// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
      onClick={[Function]}
    >
      cardText1
    </div>
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
      onClick={[Function]}
    >
      cardText2
    </div>
  </div>
</div>
`;
